### PR TITLE
Feature/acq 976/977 code org update

### PR DESCRIPTION
**Pull request by dmantonyuk**
In [code.org/about](http://code.org/about). Added the following section between “A unifying approach in a divided world” section and “Our commitment to accessibility”.
Remake [code.org/about](http://code.org/about) for the translation system so the page can be translated.
## Links

https://codedotorg.atlassian.net/browse/ACQ-977
https://codedotorg.atlassian.net/browse/ACQ-976

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
About Us - Code org

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -8449,6 +8449,62 @@
...
+  about_page_section_specific_description_7: "[Sign up to receive status updates](%{sign_up_url}) about progress in the K-12 computer science movement and about the work of Code.org. Or follow Code.org on social media:"
+  about_page_section_specific_description_8: "Body: Over 40% of our website traffic comes from outside of the United States and that number continues to climb. In order to expand global access to computer science, our team works closely with [more than 100 international partners](%{partners_url}), helping them to promote the Hour of Code, advocate for policy change, and train teachers. We are making computer science part of the international education discourse by partnering with ministries of education from around the world and working with international organizations like the Organization for Economic Co-operation and Development and the United Nations Educational, Scientific and Cultural Organization."
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Remove this: `Body: `

**Review by kelbyhawn — COMMENTED** [review]
One small copy fix needed, but otherwise looks great 🙌

**Review by kelbyhawn — APPROVED** [review]

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -8449,6 +8449,62 @@
...
+  about_us_page_section_news_description: "See all past [news and announcements](%{news_url})"
+  about_us_page_section_table: "Our goals and metrics"
+  about_us_page_section_table_header_1: "Code.org Goal"
+  about_us_page_section_table_header_2: "Accomplishment"
+  abous_page_section_table_cell_1_1: "Improve diversity in CS *(US students)*"
+  abous_page_section_table_cell_1_2: "45% of Code.org students are young women, 50% are students from marginalized racial and ethnic groups, and 46% of US students are in high needs schools. Read more about [our approach to diversity.](%{diversity_url})"
+  about_page_section_table_cell_2_1: "Inspire students"
+  about_page_section_table_cell_2_2: "Tens of millions have tried the [Hour of Code](%{hour_url}). (426,370,561 served. 49% female)"
+  about_page_section_table_cell_3_1: "Reach classrooms"
+  about_page_section_table_cell_3_2: "591,636 teachers have signed up to teach our intro courses on [Code Studio](%{code_studio_url}) and 19,177,297 students are enrolled."
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:8468]: Was it intentional that we now hardcode these numbers whereas [before](https://github.com/code-dot-org/code-dot-org/pull/53905/files#diff-cf71ff6d0ef1f7e19a35ce8b8b95efca41ddb40e534322514c3e24cb806419afL82) this change, we generated them programmatically?  And we translate these hardcoded values, too?

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 3]
```diff
@@ -8449,6 +8449,62 @@
...
+  about_us_page_section_title_1: "Over 80 million students and 2 million teachers on Code.org"
+  about_us_page_statistic_figcaption: "Code.org participation from the 2021-2022 school year. [See disaggregated data.](%{data_url})"
+  about_us_page_statistic_description: "Code.org increases diversity in computer science by reaching students of all backgrounds where they are — at their skill-level, in their schools, and in ways that inspire them to keep learning. The vast majority of the students on Code.org are from student groups historically underrepresented in computer science. Increasing diversity in computer science is foundational to our work, and [we encourage you to read more about our efforts](%{diversity_url})."
+  about_us_page_section_title_2: "Code.org in the News"
+  about_us_page_section_news_description: "See all past [news and announcements](%{news_url})"
+  about_us_page_section_table: "Our goals and metrics"
+  about_us_page_section_table_header_1: "Code.org Goal"
+  about_us_page_section_table_header_2: "Accomplishment"
+  abous_page_section_table_cell_1_1: "Improve diversity in CS *(US students)*"
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:8463]: It seems rather fragile to name these keys based on cell positions.  What happens if we want to insert a row?  

It seems things would be more readable and flexible if these key names described the content.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 2 above)
Comment (breville) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:8468]: Also, these specific values seem to be the outdated "fallback" values from [here](https://github.com/code-dot-org/code-dot-org/blob/3ce0dc5be8d75191b66905b589eed070df723c8e/pegasus/helpers/properties.rb#L48-L49), which are from 2017.  

From some quick delving on production, there seemed to be an issue preventing the newer values from being retrieved, but the actual values are much higher now.

Also, these are total numbers of students and teachers, so the "intro courses" nomenclature might be out of date.

**Pull request merged**